tests/Unit/Routes/StoreRoutesTest.php

```diff
@@ -239,6 +239,59 @@ public function testVoucherManageRouteGate()
         }
     }
 
+    public function testVoucherManagerUpdateGate()
+    {
+        // Create a random registration with our centre.
+        $registration = factory(Registration::class)->create([
+            "centre_id" => $this->centre->id,
+        ]);
+
+        $route = URL::route('store.registration.voucher-manager', [ 'registration' => $registration->id ]);
+        $put_route = URL::route('store.registration.vouchers', [ 'registration' => $registration->id ]);
+
+        // I can call a PUT on a page as a user
+        Auth::logout();
+        $this->actingAs($this->centreUser, 'store')
+            ->visit($route)
+            ->call(
+                'PUT',
+                $put_route,
+                [] // should erase the vouchers.
+            );
+        $this->assertResponseStatus(302);
+
+        // I can call put on a page as a neighbor
+        Auth::logout();
+        $this->actingAs($this->neighborUser, 'store')
```

Comment:
Neighbour user is same sponsor as the centre user which means they can act on each other's registrations?

Reply:
Yep!